build.sbt

```diff
@@ -91,3 +95,85 @@ lazy val core = project
     packageName := s"${name.value}-$daffodilVer"
   )
   .enablePlugins(commonPlugins: _*)
+  .dependsOn(sbtXjcProject)
+  .aggregate(sbtXjcProject)
+
+lazy val sbtXjcProject = project
```

Comment:
Question: I'm not sure if this needs to be a separate sbt project. Was there a particular reason why it was done this way?

I can see if locally I can move this stuff into the core project.